test/e2e/node/pods.go

```diff
@@ -46,8 +46,7 @@ var _ = SIGDescribe("Pods Extended", func() {
 		BeforeEach(func() {
 			podClient = f.PodClient()
 		})
-		// Flaky issue #36821.
-		framework.ConformanceIt("should be submitted and removed  [Flaky]", func() {
+		framework.ConformanceIt("should be submitted and removed", func() {
```

Comment:
this still looks flaky... it's setting up a watch prior to pod creation, then waiting for the pod to run, get deleted, then terminated, then deleted by the kubelet... the watch could easily time out or exceed the compaction window before that happens. the changes in https://github.com/kubernetes/kubernetes/pull/50102 seem necessary to get a robust watch that can be re-established to make this test non-flaky